### PR TITLE
fix: add chat, qna

### PR DESCRIPTION
**Pull request by ArshdeepGrover**

### CODE REVIEW

**Comment by arpansac** [comment]
QnA not working, please fix

File: projects/commudle-admin/src/app/feature-modules/public-events/components/session-page/session-page-video/session-page-video.component.html [hunk 1]
```diff
@@ -46,7 +46,12 @@
       ></app-video-stream>
     </div>
     <div
-      *ngIf="currentUser && ['commudle', 'youtube'].includes(embeddedVideoStream.source) && interactionWindows"
+      class="user-interactions"
+      [ngClass]="{ 'user-interactions-video ': !embeddedVideoStream }"
+      *ngIf="
+        !embeddedVideoStream ||
+        (currentUser && ['commudle', 'youtube', 'other'].includes(embeddedVideoStream.source) && interactionWindows)
```
Comment (arpansac) [inline projects/commudle-admin/src/app/feature-modules/public-events/components/session-page/session-page-video/session-page-video.component.html]: fix this

File: projects/commudle-admin/src/app/feature-modules/public-events/components/session-page/session-page-video/session-page-video.component.html [hunk 2]
```diff
@@ -99,7 +105,15 @@
...
       </button>
     </div>
-    <div #interactionWindow class="session-page-window">
+    <div
+      #interactionWindow
+      class="session-page-window"
+      [ngClass]="{ 'not-embedded-video': !embeddedVideoStream }"
+      *ngIf="
+        !embeddedVideoStream ||
+        (currentUser && ['commudle', 'youtube', 'other'].includes(embeddedVideoStream.source) && interactionWindows)
```
Comment (arpansac) [inline projects/commudle-admin/src/app/feature-modules/public-events/components/session-page/session-page-video/session-page-video.component.html]: fix condition

File: projects/commudle-admin/src/app/feature-modules/public-events/components/session-page/session-page-video/session-page-video.component.ts [hunk 3]
```diff
@@ -70,7 +70,7 @@ export class SessionPageVideoComponent implements OnInit, OnChanges, AfterViewIn
   }
 
   ngAfterViewInit(): void {
-    if (['commudle', 'youtube'].includes(this.embeddedVideoStream.source)) {
+    if (this.embeddedVideoStream === undefined || ['commudle', 'youtube'].includes(this.embeddedVideoStream.source)) {
```
Comment (arpansac) [inline projects/commudle-admin/src/app/feature-modules/public-events/components/session-page/session-page-video/session-page-video.component.ts]: Fix this, use iframe and check undefined condition, use !

**Review by arpansac — CHANGES_REQUESTED** [review]

File: projects/commudle-admin/src/app/feature-modules/public-events/components/session-page/session-page-video/session-page-video.component.html [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
-  [ngClass]="{
-    fullscreen: isFullScreen,
-    'internal-link': ['commudle', 'youtube'].includes(embeddedVideoStream.source),
-    'external-link': ['external_link', 'jitsi'].includes(embeddedVideoStream.source)
-  }"
+  [ngClass]="
+    !embeddedVideoStream || ['commudle', 'youtube', 'other'].includes(embeddedVideoStream.source)
+      ? 'internal-link'
+      : 'external-link'
+  "
```
Comment (ArshdeepGrover) [inline projects/commudle-admin/src/app/feature-modules/public-events/components/session-page/session-page-video/session-page-video.component.html]: check full screen condition

**Review by arpansac — APPROVED** [review]

**Pull request merged**